frontend/src/pages/profile/styles.module.scss

```diff
@@ -0,0 +1,34 @@
+.profile {
+  width: 100%;
+  max-width: 1591px;
+  margin: 0 auto;
+}
+
+.userInfoBlock {
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+  width: 270px;
+}
+
+.avatar {
+  width: 270px;
+  height: 269px;
```

Comment:
What if you'd do it this way?
```suggestion
  width: 100%;
  height: auto;
```

Reply:
![image](https://github.com/BinaryStudioAcademy/bsa-2023-writorium/assets/142595953/507cf4b1-40c7-48e0-b9d0-6ad78cf2b0f9)

in this case, I have an ellipse but not a circle

Reply:
![image](https://github.com/BinaryStudioAcademy/bsa-2023-writorium/assets/142595953/58f24de4-e1fa-467c-a534-d9017f8e08e1)

It works with this changes